src/Private/Serverless.ps1

```diff
@@ -26,7 +26,7 @@ function Start-PodeAzFuncServer {
             $response.Headers = @{}
 
             # reset event data
-            $global:WebEvent = @{
+            $WebEvent = @{
```

Comment:
This needs to be global, otherwise Functions can't see the `$WebEvent`

Reply:
What is the difference between here and 

```powershell
function Start-PodeWebServer {
#omiss
#line 118
     try {
                while ($Listener.IsConnected -and !$PodeContext.Tokens.Cancellation.IsCancellationRequested) {
                    # get request and response
                    $context = (Wait-PodeTask -Task $Listener.GetContextAsync($PodeContext.Tokens.Cancellation.Token))

                    try {
                        try {
                            $Request = $context.Request
                            $Response = $context.Response

                            # reset with basic event data
                            $WebEvent = @{
                                OnEnd            = @()
```
`Start-PodeWebServer` is not defining $WebEvent as global

Reply:
Mostly scoping, in serverless these calls work slightly differently to the normal Web Server logic - they're called inline with the parent runspace and not in their own runspace/pool. Timers a slightly similar; unless these have `$global:` then scoping makes them `$null`